test/FunctionsV2/PlatformSpecificHelpers.FunctionsV2.cs

```diff
@@ -54,5 +55,37 @@ public static class PlatformSpecificHelpers
 
             return (JobHost)host.Services.GetService<IJobHost>();
         }
+
+        private static IWebJobsBuilder AddCorrectDurableTaskExtension(this IWebJobsBuilder builder, IOptions<DurableTaskOptions> options, string storageProvider)
+        {
+            switch (storageProvider)
+            {
+                case TestHelpers.RedisProviderType:
+                    builder.AddRedisDurableTask();
+                    break;
+                case TestHelpers.EmulatorProviderType:
+                    builder.AddEmulatorDurableTask();
+                    break;
+                case TestHelpers.AzureStorageProviderType:
+                    break;
```

Comment:
Let's add a comment here explaining that the Azure Storage provider is built-in.